transformedTests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "trypescript-rtti-transformed-tests",
+    "version": "1.0.0",
+    "type": "module",
+    "scripts": {
+        "build": "tspc --build",
+        "clean": "tspc --build --clean",
+        "test": "npm run build && jest --runInBand"
+    },
+    "dependencies": {
+        "typescript-rtti": "^0.9.6"
```

Comment:
I actually have a private (`gitignore`ed) `playground` project that is very similar to what you are doing here, so I know this can work. Set this version to `".."` to have NPM install it using file deps.